constants/roles.js

```diff
@@ -4,6 +4,7 @@ const ROLES = {
   APPOWNER: "app_owner",
   MEMBER: "member",
   ARCHIVED: "archived",
+  INDISCORD: "in_discord",
```

Comment:
I guess it will be better to use a camel cased literal rather than a snake case

Reply:
super_user and app_owner were already present so I had to follow the same rule here